pontoon/base/views.py

```diff
@@ -138,19 +142,25 @@ def locale_project(request, locale, slug):
     """Locale-project overview."""
     l = get_object_or_404(Locale, code__iexact=locale)
 
-    project = (
-        get_object_or_404(
-            Project.objects.prefetch_related('subpage_set'),
-            disabled=False,
-            slug=slug,
-            pk__in=Resource.objects.values('project')
-        )
+    projects = Project.objects.prefetch_related('subpage_set').distinct()
```

Comment:
Yep, although because I split this into two lines (and I'm not entirely sure if I had a good reason to do so now, but didn't remove it) it's not obvious. The `get_object_or_404` call filters by `resources__isnull`, and anytime you filter by a ManyToMany, you'll get a result for each of the related objects. In this case, we get as many projects as there are resources in the project. Hence, the `distinct` to avoid the duplication.